### PR TITLE
Javascipt Quiz!

### PR DESCRIPTION
**Pull request by Steventyler5**
Here's the (finally) finished JS quiz! Assuming all goes to plan, clicking the Grow a Tree button should check if each field has a value, if the number is in fact a number, if more than one character was entered in the second input, and then display one of 8 alerts depending on what inputs were incorrect. If the number is just a number and the character is just one character then a tree should print to the console. The growATree() function isn't exactly pretty, but what started as a single step if else statement kinda snowballed out of control. I probably should've gone with a switch statement but I figured as long as it works I'm probably good to go. Similarly, my for loop is an eyesore. I roughed everything out in a notebook before I wrote the code to keep track of variables, but I didn't think to transfer those into comments.  If whoever ends up reviewing this would like me to do so, I'll gladly go back and add comments explaining what the less descriptive variable names were for. This counts as a "descriptive PR message", right? 
TL;DR - The javascript is a poorly documented eyesore, but I'm hoping it's forgivable as long as it works :+1:

### CODE REVIEW

**Comment by Steventyler5** [comment]
I completely misunderstood that requirement, I thought it was an either/or situation. Added a second function to handle the enter key press in either text inout and indented the last two lines of the growATree function
